resources/nheko.appdata.xml.in

```diff
@@ -37,10 +36,6 @@
     <content_attribute id="social-chat">intense</content_attribute>
     <content_attribute id="social-audio">intense</content_attribute>
   </content_rating>
-  <custom>
-    <value key="Purism::form_factor">workstation</value>
-    <value key="Purism::form_factor">mobile</value>
-  </custom>
```

Comment:
Do we actually need to remove this? Afaik this is required for some app stores to actually list stuff properly as mobile compatible.

Reply:
Even GNOME apps are removing it: https://gitlab.gnome.org/GNOME/epiphany/-/commit/20e4fda72fd363060d9bd7db68a8abae22e2ed4c (so I guess it's basically never used anymore)

Reply:
Hm, I remember some mobile app stores not showing our app as mobile compatible without this... And the warning message sounds incorrect, it should be absolutely valid to have such custom keys... But I guess the flatpak won't build otherwise on flathub?